[BUG] Dpdata failed to process `abacus/md` (the `develop` branch of `abacus`)

### Bug summary

The energy keyword in `running_md.log` have changed from `final_etot_is` to `#TOTAL ENERGY#` in the  `develop` branch of `abacus`, so `dpdata/abacus/md.py` cannot process the log files successfully.

### dpdata Version

1.0.1

### Input Files, Running Commands, Error Log, etc.

sys = dpdata.LabeledSystem("../",fmt="abacus/pw/md")

 File "/home/dpdata/abacus/md.py", line 188, in get_frame
 energy = get_energy(outlines, ndump, dump_freq)
 File "/home/dpdata/abacus/md.py", line 150, in get_energy
 assert ndump == len(energy), (
AssertionError: Number of total energies in running_md.log = 0. Number of frames in MD_dump = 20. Please check.

### Steps to Reproduce

Just use a `running_md.log` of `develop` branch of abacus to reproduce.

### Further Information, Files, and Links

_No response_

--- a/dpdata/abacus/md.py
+++ b/dpdata/abacus/md.py
@@ -135,7 +135,7 @@ def get_energy(outlines, ndump, dump_freq):
     energy = []
     nenergy = 0
     for line_idx, line in enumerate(outlines):
-        if "final etot is" in line:
+        if "final etot is" in line or "#TOTAL ENERGY#" in line:
             if nenergy % dump_freq == 0:
                 energy.append(float(line.split()[-2]))
             nenergy += 1

--- a/dpdata/abacus/relax.py
+++ b/dpdata/abacus/relax.py
@@ -94,7 +94,7 @@ def get_coords_from_log(loglines, natoms):
                     list(map(lambda x: float(x) * a0, loglines[i + k].split()[0:3]))
                 )
 
-        elif line[1:14] == "final etot is":
+        elif line[1:14] == "final etot is" or "#TOTAL ENERGY#" in line:
             # add the energy for previous structures whose SCF is not converged
             while len(energy) < len(coords) - 1:
                 energy.append(np.nan)

